src/renderer/components/chat/ChatListContextMenu.tsx

```diff
@@ -110,7 +109,6 @@ export function useChatListContextMenu(): {
         openLeaveChatDialog(openDialog, chatListItem.id)
       const onBlockContact = () =>
         openBlockFirstContactOfChatDialog(openDialog, chatListItem)
-      const onMuteChat = () => openMuteChatDialog(openDialog, chatListItem.id)
```

Comment:
We probably want to use this change in the `ThreeDotMenu` component as well and then I think we can remove the then unused `openMuteChatDialog` method and then dialog component altogether?

Reply:
ah now I get what you mean, sure that's a good idea.

Reply:
I did the ThreeDotMenu part, I also did a quick fix for the submenus that don't fit on the right side of the screen, but since we don't have any that might not fit on left side I didn't bother with that for now and I feel like it's a bit more complicated issue if we have couple of sublevel that don't fit, so I just went for the easiest thing.